ui/src/app/utils/authentication.tsx

```diff
@@ -7,161 +8,192 @@ import { userApi } from 'app/services/swagger-fetch-clients';
 import { AnalyticsTracker, setLoggedInState } from 'app/utils/analytics';
 import { LOCAL_STORAGE_KEY_TEST_ACCESS_TOKEN } from 'app/utils/cookies';
 import { navigateSignOut } from 'app/utils/navigation';
-import {
-  AuthStore,
-  authStore,
-  serverConfigStore,
-  useStore,
-} from 'app/utils/stores';
+import { authStore, serverConfigStore } from 'app/utils/stores';
 import { delay } from 'app/utils/subscribable';
+import { User, WebStorageStateStore } from 'oidc-client-ts';
 
 import { DEMOGRAPHIC_SURVEY_SESSION_KEY } from './constants';
 
-declare const gapi: any;
+const GOOGLE_BILLING_SCOPE = 'https://www.googleapis.com/auth/cloud-billing';
 
-// for e2e tests: provide your own oauth token to obviate Google's oauth UI
```

Comment:
is this comment still valid?

Reply:
Good point, re-added. I had moved this comment to `ui/src/app/services/setup.ts`, but I think it's valid to duplicate the comment instead since the connection between the files isn't obvious.